client/me/help/help-contact/index.jsx

```diff
@@ -420,6 +427,11 @@ const HelpContact = React.createClass( {
 		return isHappychatAvailable && olark.isUserEligible;
 	},
 
+	shouldUseDirectly: function() {
+		const isEn = i18n.getLocaleSlug() === 'en';
```

Comment:
There's a `currentUserLocale` prop which gets [passed in via Redux](https://github.com/Automattic/wp-calypso/blob/c01e34173bfde6cef399ed8f1281fc0c535beb79/client/me/help/help-contact/index.jsx#L657) and could be used here. That should also better handle the (albeit rare) cases where the user's locale changes.